capa/ida/plugin/item.py

```diff
@@ -210,14 +214,16 @@ class CapaExplorerFunctionItem(CapaExplorerDataItem):
 
     fmt = "function(%s)"
 
-    def __init__(self, parent, location, can_check=True):
+    def __init__(self, parent: CapaExplorerDataItem, location: Address, can_check=True):
         """initialize item
 
         @param parent: parent node
         @param location: virtual address of function as seen by IDA
         """
+        assert isinstance(location, AbsoluteVirtualAddress)
+        ea = int(location)
```

Comment:
note that we assume that IDA only works with absolute addresses. that is, it doesn't work with .NET tokens, etc. as of today, this is true. if we change the behavior of the plugin in the future, we may have to update these assumptions. thats why i used an `assert` here.